packages/core/src/transports/types.ts

```diff
@@ -29,6 +29,13 @@ export type UTXOAPISchema = [
       hasMore?: boolean
     }
   },
+  {
+    Method: 'getTransactionFee'
```

Comment:
We should add `getTransactionFee` to UTXOAPISchemaMethods so we can exclude it safely from RPC calls